09_elements.html

```diff
@@ -289,4 +289,202 @@ <h2 id="element-location-strategies">Element Location Strategies</h2>
+     run the following substeps:
+
+     <ol>
+      <li>Apply the <a>element displayed</a> algorithm to <var>parent</var>.
+
+      <li>If the return value is false, abort these steps and return that value.
+
+      <li>If <var>parent</var> is a block element box and the
+       computed values of either <code>overflow-x</code>
+       or <code>overflow-y</code> is "hidden":
+
+      <ol>
+       <li>Let <var>parent dimensions</var> be the <code>DOMRect</code>
+        that is the first element of the <code>DOMRectList</code>
+        array returned by calling
+        <code><a href=http://www.w3.org/TR/cssom-view/#dom-range-getclientrects>getClientRects</a></code>
+        on <var>parent</var>.
+
+       <li>Let <var>element dimensions</var> be the <code>DOMRect</code>
+        that is the first element of the <code>DOMRectList</code>
+        array returned by calling
+        <code><a href=http://www.w3.org/TR/cssom-view/#dom-range-getclientrects>getClientRects</a></code>
+        on <var>element</var>.
+
+       <li>Let <var>parent style</var> be the computed style of <var>parent</var>.
+
+       <li>Return false if any the following conditions evaluate to false:
+
+       <ul>
+        <li><var>element dimension</var>'s <code>top</code>
+         is less than (<var>parent dimension</var>'s <code>bottom</code>
+         − the <a title="normalize style pixel values to floating point">normalized style pixel float value</a>
+         of <var>parent style</var>'s <code>borderBottomWidth</code>).
+
+        <li><var>element dimension</var>'s <code>bottom</code>
+         is less than (<var>parent dimension</var>'s <code>top</code>
+         − the <a title="normalize style pixel values to floating point">normalized style pixel float value</a>
+         of <var>parent style</var>'s <code>borderTopWidth</code>).
+
+        <li><var>element dimension</var>'s <code>left</code>
+         is less than (<var>parent dimension</var>'s <code>right</code>
+         − the <a title="normalize style pixel values to floating point">normalized style pixel float value</a>
+         of <var>parent style</var>'s <code>borderRightWidth</code> casted as a float).
+
+        <li><var>element dimension</var>'s <code>right</code>
+         is less than (<var>parent dimension</var>'s <code>left</code>
+         − the <a title="normalize style pixel values to floating point">normalized style pixel float value</a>
+         of <var>parent style</var>'s <code>borderLeftWidth</code> casted as a float).
+      </ul>
+
+      <li>Run step 10 on the parent elements of <var>parent</var>, if any.
+     </ol>
+    </ol>
+
+    <li>Return true.
+   </ol>
+
+   <p>If the steps above return true, then the element MUST also
+    be <a href=#interactable-elements>interactable</a> should it
+    meet the other criteria for being interactable.
+    If any part of the <code>BODY</code> can be brought into the current viewport,
```

Comment:
What is 'any part of the BODY' ?
It may be just me, but I think that is ambiguous, vague and does not signify a thing specific enough for the purpose of this document.

Next, 'can be brought into the current viewport' is, I think, broad and unspecific. For example, one may be able to bring a 'part of the BODY' into the current viewport by modifying current state of the DOM (hiding, showing some elements, changing their size and other properties). I think this sentence didn't mean to 'allow' such means for bringing 'any part of the BODY' into the current viewport; it should be changed to actually rule out such things.